### PR TITLE
fix: update OSCAL version

### PR DESCRIPTION
**Pull request by CloudBeard**
Updating OSCAL config files from 1.0.4 to 1.1.1 

BB Upstream is in middle of update as well.

Also fixed a name issue with a github action job I found while testing local.

Also moved Velero OSCAL

### CODE REVIEW

**Review by mjnagel — APPROVED** [review]
:shipit:

**Comment by CloudBeard** [comment]
I think ive broken the test-release and test-upgrade jobs.

File: .github/workflows/oscal/config-dubbd-aws-oscal-component.yaml [hunk 1]
```diff
@@ -11,6 +11,8 @@ metadata: # OSCAL-compliant metadata
           rel: website
       type: organization
 components: # Define paths on the local filesystem or remote paths in git repositories to OSCAL component definition files
+  local:
+    - name: aws/dubbd-aws/velero-oscal-component.yaml
```
[the remark this answers is not in the document]
Comment (CloudBeard) [reply .github/workflows/oscal/config-dubbd-aws-oscal-component.yaml:15]: Yes but we can move it if there is a better place for velero-oscal-component.yaml

**Review by MxNxPx — APPROVED** [review]
:ship: :it:

**Pull request merged**